lib/ovirt/template.rb

```diff
@@ -152,9 +152,7 @@ def build_clone_xml(options)
             end
           end
 
-          xml.os(:type => options[:os_type] || os_type) do
-            xml.boot(:dev => 'hd')
-          end
+          xml.os(:type => options[:os_type] || os_type)
```

Comment:
@gmcculloug @bdunne Yes, indeed. When the boot device(s) are missing, the default is correctly taken from the template - even the blank one; in which case it is `hd`. Also tested it just to be sure :+1: